internal/cmd/skupper/version/kube/version.go

```diff
@@ -58,10 +61,27 @@ func (cmd *CmdVersion) ValidateInput(args []string) []error {
 }
 
 func (cmd *CmdVersion) InputToOptions() {
+
+	mapRunningPods := make(map[string]string)
+
+	if cmd.KubeClient != nil {
+		// search for running pods in all namespaces
+		runningPodList, err := cmd.KubeClient.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{LabelSelector: "app.kubernetes.io/part-of=skupper"})
+		if err != nil {
```

Comment:
I think the network-observer and network-observer-prometheus does not have the label 
app.kubernetes.io/part-of=skupper  
but it does have 
app.kubernetes.io/part-of: skupper-network-observer